### DISCUSSION TITLE
MACH tutorial wing error

### DISCUSSION THREAD

**Discussion by Ryupuriwatt** [mdolab/dafoam#543] [Tutorials]
![image](https://github.com/mdolab/dafoam/assets/115110434/82fbac3b-8c67-4ef7-b498-92fb27be50d0)
When I try to run this tutorial I get the [44604,1],0]: A high-performance Open MPI point-to-point messaging module
was unable to find any relevant network interfaces:

Module: OpenFabrics (openib)
  Host: 467a5c745afa

Another transport will be used instead, although this may result in
lower performance.

NOTE: You can disable this warning by setting the MCA parameter
btl_base_warn_component_unused to 0.
So I would like to know how to fix this error. Right now I use latest version of dafoam from docker

**Comment by friedenhe** [comment]
I ran this case without a problem. Which version of the Docker image you are using? The message you posted seems to be a pretty common MPI warning instead of an error.
